packages/lib-classifier/webpack.dev.js

```diff
@@ -17,9 +17,9 @@ const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
 module.exports = {
   devServer: {
     allowedHosts: [
-      'localhost',
```

Comment:
I think I copied this from PFE.

Reply:
if this change is not necessary to the docker configuration, I'd prefer that it is left in since it's unclear why it is being changed. I'm not sure why PFE doesn't include it in the allowed hosts.

Reply:
Feel free to add it back and merge this if it still works. The dev server doesn’t run on localhost inside the container. It runs on 0.0.0.0:8080